packages/readme.md

```diff
@@ -12,7 +12,7 @@ To control the build behaviour of your package, use variables in the top-down or
 | Variable    | Default | Required |Description |
 |-------------|---------|----------|------------|
 | PKG_NAME    | -       | yes | Name of the packaged software application. Should be lowercase |
-| PKG_VERSION | -       | yes | Version of the packaged software application |
+| PKG_VERSION | -       | yes | Version of the packaged software application<br />If the version is a githash, please use the full githash, not the abbreviated form. |
```

Comment:
Please use a period to denote the end of a sentence. Browsers can handle flow over multiple lines so there is no need to artificially force a line break.